docs/common-concepts/event-bus.mdx

```diff
@@ -0,0 +1,185 @@
+
+Writing a guide and a concept article are not mutually exclusive. Instead,
+you should, in most cases, write both, where the concept gives a brief overview
+of what it is, and the guide is a longer article explaining best practices on using
+it.
+
+E.g.:
+Concept: Event Bus
+Guide: Interacting with the Event Bus
+
+
+=== Writing tips:
+- Write in the present tense
+- Use neutral pronouns (they/them instead of he/she and him/her)
+- Be respectful to everyone
+- Be aware of the potential for cultural misunderstandings
+-->
+
+<!-- Relevant imports: -->
+
+import { Reference, Image } from '/components';
+
+<!-- Brief Description (1-3 lines) of the concept: -->
+
+The Event Bus is, basically, the nervous system of the Telestion Ecosystem. The
+ecosystem uses the Event Bus for the communication between its various
+components.
+
+## Video version
+
+You can find a video version of this concept's explanation here:
+
+<Reference to="https://www.youtube.com/watch?v=X92A77OdEg0">
+	Event Bus | Telestion Concepts (Video)
+</Reference>
+
+## Messages
+
+At the core of the Event Bus, there are so-called _Messages_. Messages are data
+packets that you can send to a specific channel. Then, all components monitoring
+that channel receive you message.
+
+## Channels and Addresses
+
+A _Channel_ is defined by its _Address_ (or _Channel Address_), which is a
+string (text) and essentially serves as the channel's name.
+
+<Image
+	invertible
+	src="/static/img/drawio-diagrams/vertx/eventbus.drawio.svg"
+	center
+/>
+
+The address typically describes the channel's content (e.g., `raw-telemetry`,
+`telecommand-out`, etc.).
+
+:::tip Mental model
+
+As a mental model, you can compare the event bus to radio communication, e.g.,
+in Air Traffic Control:
+
+| Concept in Radio Communication | Corresponding concept in the Event Bus |
+| ------------------------------ | -------------------------------------- |
+| Radio Transmission             | Message                                |
+| Frequency                      | Address                                |
+
+:::
+
+## Data Pipeline
+
+Through the combination of more than one component
+([Verticles](/application/concepts/verticle/) on the _Application_ side) sending
+each other messages and performing small, atomic tasks, you can create a sort of
+data pipeline that's both easier to think about (as individual components only
+need to perform smaller tasks) and more adjustable (since you can just plug
+components together in different ways):
+
+<Image
+	invertible
+	src="/static/img/drawio-diagrams/vertx/data-pipeline.drawio.svg"
+	center
+/>
+
+## Requests
+
+The event bus also allows for so-called _Requests_.
+
+This enables _Request Messages_ which another component can then respond to.
+This is most useful when dealing with data that you don't need constantly, but
+only on specific occasions:
+
+<Image
+	invertible
+	src="/static/img/drawio-diagrams/vertx/requests.drawio.svg"
+	center
+/>
```

Comment:
We should change this schematic a bit, because it suggests that you can define the response address, which is not possible. Vert.x generates a one-time response address for this specific message.

Maybe move both arrows between the verticles together and hide the reponse address?

Reply:
I partially agree. I want to communicate how this gets handled (i.e., the response address), since this makes the concept easier for developers to understand (since the way it works makes sense: "please call me back on that phone number" is a good mental model).

But I will add a note about this address being auto-generated here.